docs/gettingstarted.md

```diff
@@ -0,0 +1,58 @@
+
+# Pre-req
+
+* Linux host with Docker 19.x or newer with Docker Swarm enabled
+    [Getting Started](https://docs.docker.com/get-started/)
+* A Splunk index for metrics typically "em_metrics"
+* One or more Splunk indexes for events collected by SC4S
+* Splunk HTTP event collector enabled with a token dedicated for SC4S
+    * [Splunk Enterprise](http://dev.splunk.com/view/event-collector/SP-CAAAE6Q)
+    * [Splunk Enterprise Cloud](http://docs.splunk.com/Documentation/Splunk/7.3.1/Data/UsetheHTTPEventCollector#Configure_HTTP_Event_Collector_on_managed_Splunk_Cloud)
+* A network load balancer (NLB) configured for round robin. Note: Special consideration may be required when more advanced products are used. The optimal configuration of the load balancer will round robin each post (not each connection)
```

Comment:
"round robin each post"
Should this be "port" ?

Reply:
I believe "post" is correct; the round robin applies to each http post request made by the client.

Reply:
no post,  a connection and accept multiple posts while open but with ALB the RR is at the post level

Reply:
A network load balancer should _not_ be required; internal syslog-ng LB works well and should be encouraged if the customer situation will allow it.  This will allow proper, keep-alive sessions directly to each indexer to be configured, at the expense of some overhead in properly maintaining the indexer list.

Reply:
I agree that's actually true if we allow non LB use then we should require DNS a-records where "hec-splunk.company.com" has all the entries for all the indexers. I don't think we should maintain the list in docker (I already tested this it works)